builtInFunctions/esdtDataStorage.go

```diff
@@ -363,7 +363,6 @@ func (e *esdtDataStorage) saveESDTMetaDataToSystemAccount(
 	if len(currentSaveData) == 0 && e.flagSendAlwaysEnableEpoch.IsSet() {
 		esdtDataOnSystemAcc.Properties = nil
 		esdtDataOnSystemAcc.Reserved = []byte{1}
-		esdtDataOnSystemAcc.Value.Set(esdtData.Value)
```

Comment:
this line was added in https://github.com/ElrondNetwork/elrond-vm-common/commit/7346e85eca473e7161a6d3f3de028bb8ebe2d163 with an esdtStorageHandler.AddToLiquiditySystemAcc, similar to the one added in this PR. 
is it ok to be removed?

Reply:
Yes. It is ok

Reply:
ok!